MOS-API.md

```diff
@@ -109,249 +113,260 @@ See mos_api.asm for implementation
+| 4-7 | Reserved for future use (for future compatibility, ensure these are set to zero) |
+
+\* Support for editor control flags was added in Console8 MOS 2.2.0.  Prior to this the only documented values for `E` were 0 and 1 to indicate whether the buffer should be cleared.
 
 Returns:
-- A: Key that was used to exit the input loop (CR=13, ESC=27)
+- `A`: Key that was used to exit the input loop (CR=13, ESC=27)
 
-### 0x0A: mos_fopen
+### `0x0A`: mos_fopen
 
 Get a file handle
 
 Parameters: 
 
-- HL(U): Address of filename (zero terminated)
-- C: Mode
+- `HL(U)`: Address of filename (zero terminated)
+- `C`: Mode
 
 Returns:
-- A: Filehandle, or 0 if couldn't open
+- `A`: Filehandle, or 0 if couldn't open
 
 Mode can be one of: fa_read, fa_write, fa_open_existing, fa_create_new, fa_create_always, fa_open_always or fa_open_append
 
 NB: If you open the file using mos_fopen, you must close it using mos_fclose, not ffs_api_fclose
 
-### 0x0B: mos_fclose
+### `0x0B`: mos_fclose
 
 Close a file handle
 
 Parameters: 
 
-- C: Filehandle, or 0 to close all open files
+- `C`: Filehandle, or 0 to close all open files
 
 Returns:
 
-- A: Number of files still open
+- `A`: Number of files still open
 
-### 0x0C: mos_fgetc
+### `0x0C`: mos_fgetc
 
 Get a character from an open file
 
 Parameters: 
 
-- C: Filehandle
+- `C`: Filehandle
 
 Returns:
-- A: Character read
-- F: C set if last character in file, otherwise NC (MOS 1.04 or greater)
+- `A`: Character read
+- `F`: C set if last character in file, otherwise NC (MOS 1.04 or greater)
 
-### 0x0D: mos_fputc
+### `0x0D`: mos_fputc
 
 Write a character to an open file
 
 Parameters: 
 
-- C: Filehandle
-- B: Character to write
+- `C`: Filehandle
+- `B`: Character to write
 
 Returns: None
 
-### 0x0E: mos_feof
+### `0x0E`: mos_feof
 
 Check for end of file
 
 Parameters: 
 
-- C: Filehandle
+- `C`: Filehandle
 
 Returns:
-- A: 1 if at end of file, otherwise 0
+- `A`: 1 if at end of file, otherwise 0
 
-### 0x0F: mos_getError
+### `0x0F`: mos_getError
 
 Copy an error string to a buffer
 
 Parameters: 
 
-- E: The error code
-- HL(U): Address of buffer to copy message into
-- BC(U): Size of buffer
+- `E`: The error code
+- `HL(U)`: Address of buffer to copy message into
+- `BC(U)`: Size of buffer
 
 Returns: None
 
-### 0x10: mos_oscli
+### `0x10`: mos_oscli
 
 Execute a MOS command
 
 Parameters: 
 
-- HLU: Pointer the the MOS command string
-- DEU: Pointer to additional command structure
-- BCU: Number of additional commands
+- `HLU`: Pointer the the MOS command string
+- `DEU`: Pointer to additional command structure
+- `BCU`: Number of additional commands
 
 Returns:
 
-- A: MOS error code
+- `A`: MOS error code
 
-### 0x11: mos_copy
+### `0x11`: mos_copy
 
 Copy a file on the SD card
 
 Parameters: 
 
-- HL(U): Address of filename1 (zero terminated)
-- D(E)U: Address of filename2 (zero terminated)
+- `HL(U)`: Address of filename1 (zero terminated)
+-` D(E)U`: Address of filename2 (zero terminated)
```

Comment:
Looks like a typo

Reply:
Yep - well spotted - probably a mistake in the original docs

I’ll fix

also need to document the hotkey command.